java/src/main/java/com/codurance/training/tasks/service/ITaskViewService.java

```diff
@@ -0,0 +1,7 @@
+package com.codurance.training.tasks.service;
+
+public interface ITaskViewService {
+    void viewByDate();
+    void viewByDeadline();
+    void viewByProject();
```

Comment:
Instead of creating 3 methods here just keep one method...and create 3 different classes (ViewByDate,ViewByDeadline, ViewByProject) which implements this interface. 

This is to ensure the open closed principle like if in case if we need to add viewByTask then instead of modifying this interface we can simply implement in ViewByTask class(new class)

Reply:
got it, will update :+1: